build.gradle

```diff
@@ -100,6 +100,7 @@ subprojects {
     if (project.hasProperty('mvnlocal')) {
       mavenLocal()
     }
+    mavenLocal()
```

Comment:
@ljacomet : I approved the changes because I opened PR #1676 that includes your changes plus mine for the new tc-platform release and m&m stuff (since I depend on this PR).
I fixed the `mavenLocal()` thing in it.